test/ProgressOnderwijsUtils.Tests/Data/ReadJsonTest.ReadJson_can_read_all_known_used_column_types_from_the_db.approved.txt

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "ReadJsonTestId": 1,
+    "BitColumn": true,
+    "IntColumn": 2147483647,
+    "BigIntColumn": 9223372036854775807,
+    "DecimalColumn": 0.99,
+    "FloatColumn": 1.234,
+    "DateColumn": "2008-04-01T00:00:00",
+    "DateTimeColumn": "2023-05-06T16:13:55",
+    "DateTime2Column": "0001-02-03T04:05:06.007",
+    "CharColumn": "x",
+    "VarCharColumn": "xyz",
+    "NCharColumn": "p",
+    "NVarCharColumn": "pqr",
+    "BinaryColumn": "\u0001\u0002\u0003\u0000\u0000\u0000\u0000\u0000",
+    "UniqueIdentifierColumn": "82dbee37-3af8-46f2-a403-ae0a1950bc6e",
+    "RowVersionColumn": "\u0000\u0000\u0000\u0000\u0000\u0000\u0007\uFFFD"

```

Comment:
Or should be better present this an an integer?

Reply:
There is no `long` i.e. 64-bit int in javascript, and while json doesn't really say either way, I think it's probably unwise to depend on integers of more than the double-precision mantissa's 53 bits.  It might if none of our consumers are javascript clients, but who knows...